diff --git a/node_modules/people-names/data/additional-names.json b/node_modules/people-names/data/additional-names.json
new file mode 100644
index 0000000..76df97f
--- /dev/null
+++ b/node_modules/people-names/data/additional-names.json
@@ -0,0 +1,4 @@
+[
+  "Biden",
+  "Trump"
+]
\ No newline at end of file
diff --git a/node_modules/people-names/lib/index.js b/node_modules/people-names/lib/index.js
index c381a22..dac6422 100644
--- a/node_modules/people-names/lib/index.js
+++ b/node_modules/people-names/lib/index.js
@@ -24,6 +24,8 @@ var femaleHumanNamesEs = require('../data/female-human-names-es.json');
 var maleHumanNamesEs = require('../data/male-human-names-es.json');
 var allHumanNamesEs = [];
 
+var additionalNames = require('../data/additional-names.json');
+
 var allNames = [];
 
 var allHumanNamesEn = femaleHumanNamesEn.concat(maleHumanNamesEn);
@@ -32,7 +34,7 @@ var allHumanNamesFr = femaleHumanNamesFr.concat(maleHumanNamesFr);
 var allHumanNamesDe = femaleHumanNamesDe.concat(maleHumanNamesDe);
 var allHumanNamesEs = femaleHumanNamesEs.concat(maleHumanNamesEs);
 
-allNames = allHumanNamesEn.concat(allHumanNamesIt).concat(allHumanNamesFr).concat(allHumanNamesDe).concat(allHumanNamesEs);
+allNames = allHumanNamesEn.concat(allHumanNamesIt).concat(allHumanNamesFr).concat(allHumanNamesDe).concat(allHumanNamesEs).concat(additionalNames);
 
 // femaleHumanNamesEn.forEach(function (el, i) {
 // 	if ((maleHumanNamesEn[i] != undefined) && (maleHumanNamesEn[i] != 'undefined')) {
